src/main/java/pro/belbix/ethparser/swaggerConfig/HarvestControllerAPI.yaml

```diff
@@ -0,0 +1,353 @@
+openapi: 3.0.3
```

Comment:
we don't need static swager config since we have an open api UI implementation

Reply:
corrected everything except:

"we don't need a static swager configuration as we have a public API implementation."

I still believe that this will help, and also if new methods appear in the controller, they will be done in the same way as the previous ones, from here there will always be up-to-date information. It will also be more clear for new people what is happening in the methods, but the choice is still yours. It is not a fact that the yaml file will always contain up-to-date information, because someone will have to add it there when changes are made.